assets/json/assets.json

```diff
@@ -220,7 +232,55 @@
                                 "anchor":   [0.5,0.5],
                                 "scale":    0.4
                             }
-                        }
+                        },
+                        "reconnect": {
+							"type"      : "Node",
+				            "format"    : {
+				                "type" : "Anchored"
+				            },
+				            "children":  {
+				            	"dim" : {
+				            		"type": "Image",
+				                    "data": {
+				                        "texture":  "reconnect_dim",
+				                        "position": [0,0],
+				                        "anchor":   [0.5,0.5],
+				                        "scale":    1,
+				                        "visible":  false
```

Comment:
Why don't you just set `visible` on the parent node, so you don't have to manually specify it for all 4 of the children?

Reply:
I'll try this out